File: src/test/java/leets/leenk/domain/feed/test/ReactionTestFixture.java
package leets.leenk.domain.feed.test;

import leets.leenk.domain.feed.domain.entity.Feed;
import leets.leenk.domain.feed.domain.entity.Reaction;
import leets.leenk.domain.user.domain.entity.User;

public class ReactionTestFixture {
    public static Reaction createReaction(Feed feed, User user, long count) {
        return Reaction.builder()
                .feed(feed)
                .user(user)
                .reactionCount(count)
                .build();
    }
}

File: src/test/kotlin/leets/leenk/domain/feed/test/ReactionTestFixture.kt
package leets.leenk.domain.feed.test

import leets.leenk.domain.feed.domain.entity.Feed
import leets.leenk.domain.feed.domain.entity.Reaction
import leets.leenk.domain.user.domain.entity.User

object ReactionTestFixture {
    fun createReaction(
        feed: Feed,
        user: User,
        count: Long,
    ): Reaction =
        Reaction(
            feed = feed,
            user = user,
            reactionCount = count,
        )
}
